### DISCUSSION TITLE
RealWorld example apps?

### DISCUSSION THREAD

**Discussion by bakbuz** [AvaloniaUI/Avalonia#14539] [General]
Hi guys

Is there a RealWorld example implementation for a better begin?

Thanks

https://codebase.show/projects/realworld?category=frontend&language=c%23

**Comment by thevortexcloud** [comment]
Implementation of what? Avalonia? If so, there are many. 

There are many listed here:

https://avaloniaui.net/Showcase

Most of them are open source. 

However if you are just trying to learn then there are better things to look at. The music store tutorial will cover most of the basics.

https://docs.avaloniaui.net/docs/tutorials/music-store-app/
